src/modules/keyboardmanager/ui/KeyDropDownControl.cpp

```diff
@@ -5,29 +5,34 @@
 #include "BufferValidationHelpers.h"
 #include <common\shared_constants.h>
 #include <common\keyboard_layout_impl.h>
+#include <modules\keyboardmanager\common\Helpers.h>
 
 // Initialized to null
 KeyboardManagerState* KeyDropDownControl::keyboardManagerState = nullptr;
 
-// Get keys code list depending if Disable is in dropdown
-std::vector<DWORD> KeyDropDownControl::GetKeyCodeList(bool isShortcut, bool renderDisable)
+// Get selected value of dropdown or -1 if nothing is selected
+DWORD KeyDropDownControl::GetSelectedValue(ComboBox comboBox)
 {
-    auto list = keyboardManagerState->keyboardMap.GetKeyCodeList(isShortcut);
-    if (renderDisable)
-    {
-        list.insert(list.begin(), CommonSharedConstants::VK_DISABLED);
-    }
+    auto dataContext = comboBox.SelectedValue();
+    if (!dataContext)
+        return -1;
 
-    return list;
+    auto value = winrt::unbox_value<hstring>(dataContext);
+    return stoi(to_string(value));
```

Comment:
[nit] we can use to_wstring since its from hstring.

Reply:
done